### DISCUSSION TITLE
How can I listen to the buttons on the Android TV remote control?

### DISCUSSION THREAD

**Discussion by bbhxwl** [AvaloniaUI/Avalonia#15918] [General]
How can I listen to the buttons on the Android TV remote control?

**Comment by maxkatz6** [comment]
KeyDown events are raised for the TV control as well.
It might be possible that not all possible keys are mapped, let us know if any is missing.

**Comment by 3rob3** [comment]
Key.Enter (select button on remote) doesn't seem to be mapped correctly. It comes in as Key.None with a KeyModifier.Shift for me. Up,Down,Left,Right works fine.

**Reply by maxkatz6** [reply]
Are you using 11.1 or nightly builds?

**Reply by maxkatz6** [reply]
It's only supported in 11.1 or higher

**Reply by 3rob3** [reply]
> Are you using 11.1 or nightly builds?

I'm using 11.0.11. I should switch to nightly to get this support?

**Reply by maxkatz6** [reply]
You can use 11.1 release candidate.
